gips/scripts/config.py

```diff
@@ -56,12 +56,11 @@ def create_data_variables():
     data_variables = get_data_variables()
 
     for dv in data_variables:
-        dv_model = DataVariable(**dv)
-        try:
-            dv_model.save()
-        except django.db.IntegrityError:
-            # Name was not unique
-            continue
+        name = dv.pop('name')
+        dv_model, created = DataVariable.objects.update_or_create(
+            name=name, defaults=dv
+        )
+        dv_model.save()
```

Comment:
optional:  `update_or_create` includes a `save()` so this line can be taken out:

https://docs.djangoproject.com/en/1.11/ref/models/querysets/#update-or-create

It's harmless to leave it in for now though.